pkgs/stdenv/generic/setup.sh

```diff
@@ -1508,31 +1508,20 @@ distPhase() {
 
 showPhaseHeader() {
     local phase="$1"
-    case "$phase" in
-        unpackPhase) echo "unpacking sources";;
-        patchPhase) echo "patching sources";;
-        configurePhase) echo "configuring";;
-        buildPhase) echo "building";;
-        checkPhase) echo "running tests";;
-        installPhase) echo "installing";;
-        fixupPhase) echo "post-installation fixup";;
-        installCheckPhase) echo "running install tests";;
-        *) echo "$phase";;
-    esac
+        echo "started phase: $phase"
```

Comment:
:+1: on removing the special casing of certain phases, would be fine to split this in to it's own PR and merging as.

I looked through the github search for a few of the strings and couldn't see anyone using them.